src/Experiment/src/AssemblyBuilder.cs

```diff
@@ -7,21 +7,69 @@
 using System.Text;
 using System.Threading.Tasks;
 using System.Reflection;
+using System.Reflection.Metadata.Ecma335;
+using System.Reflection.Metadata;
```

Comment:
NIT: we usually try to keep our `using` statements in alphabetical order and remove the unused ones. There is a VS context menu option that can do that for you.